libobs-opengl/gl-cocoa.m

```diff
@@ -482,15 +482,19 @@ void gl_getclientsize(const struct gs_swap_chain *swap, uint32_t *width, uint32_
 
             break;
         }
+            /* Versions of Syphon before 2dc6d31 provide IOSurfaces with pixel format 0.
+             * A pixel format of 0 means they are invalid, but in practice the IOSurfaces provided by Syphon are always
+             * 32BGRA. Historically, OBS had a bug that accidentally treated any invalid surface as 32BGRA, so those
+             * invalid surfaces were never noticed. To keep Syphon working, we need to keep this (wrong) behavior.
+             * This is suboptimal as IOSurfaces that are actually invalid (possibly coming from somewhere else) also
+             * get treated as valid 32BGRA, but it is how it is. */
+        case 0:
```

Comment:
I don't know if the indentation level here is expected or not.

Reply:
On the first line it’s forced by clang-format. I adjusted the other lines to be the same.